e2e/Makefile

```diff
@@ -2,15 +2,15 @@
 TEST_KUBERNETES_TARGET ?= current
 
 ifeq ($(TEST_KUBERNETES_TARGET),current)
-KUBERNETES_VERSION := 1.24.2
+KUBERNETES_VERSION := 1.25.3
 else ifeq ($(TEST_KUBERNETES_TARGET),prev)
-KUBERNETES_VERSION := 1.23.6
+KUBERNETES_VERSION := 1.24.7
 else ifeq ($(TEST_KUBERNETES_TARGET),prev2)
-KUBERNETES_VERSION := 1.22.9
+KUBERNETES_VERSION := 1.23.13
 endif
 export KUBERNETES_VERSION
 
-TOPOLVM_VERSION := topolvm-chart-v9.0.1
+TOPOLVM_VERSION := 31acde774d47c920ef3969b5569935761ba82834
```

Comment:
Will this value be edited when released like `topolvm-chart-vX.Y.Z`?

Reply:
We don't need to worry about it because the topolvm version will be fixed to the format like `topolvm-chart-vX.Y.Z` in the next release by the following step.
https://github.com/topolvm/pvc-autoresizer/blob/main/RELEASE.md?plain=1#L71

Also, according to the following document, setting a hash value to `TOPOLVM_VERSION ` is allowed.
https://github.com/topolvm/pvc-autoresizer/blob/main/docs/maintenance.md?plain=1#L23